pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+              var[sh][sheet_var[sheet]+'|Electricity|Coal|Brown Coal'] =var[sh][sheet_var[sheet]+'|Electricity|Coal|Brown Coal|Lignite']          
+              var[sh][sheet_var[sheet]+'|Electricity|Coal'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='coal').filter(like =country)).sum(axis=1))) + var[sh][sheet_var[sheet]+'|Electricity|Coal|Brown Coal']                                                    
+ 
+              #Energy generation : Natural gas(OCGT, CCGT, CHP, CHP CC)       
+                                                                 
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Fossil|Natural gas'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='gas CHP').filter(like =country)).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Fossil|Natural gas'] += 0.001*ct*gen_des(time( (n.links_t.p1.filter(like ='OCGT').filter(like =country)).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Fossil|Natural gas'] += 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='CCGT').filter(like =country)).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Fossil|Natural gas|CCS'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='gas CHP CC').filter(like =country)).sum(axis=1)))
+            
+              #Energy generation :Biomass (CCS)
+                                                                 
+              var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|Primary solid biomass'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='solid biomass CHP').filter(like =country)).sum(axis=1))) 
+              var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste'] = var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|Primary solid biomass']
+              var[sh][sheet_var[sheet]+'|Electricty|Biomass'] = var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste']
+            
+              var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|Primary solid biomass|CCS'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='solid biomass CHP CC').filter(like =country)).sum(axis=1)))            
+              var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|CCS'] = var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|Primary solid biomass|CCS']
+              var[sh][sheet_var[sheet]+'|Electricty|Biomass|CCS'] = var[sh][sheet_var[sheet]+'|Electricity|Solid bio and waste|CCS']            
+           
+              #Energy generation : Hydrogen
+            
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Hydrogen'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='H2 Fuel Cell').filter(like =country)).sum(axis=1)))
+
+              #Energy generation : Oil 
+
+              var[sh][sheet_var[sheet]+'|Electricity|Liquids|Fossil'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='oil-').filter(like =country)).sum(axis=1)))
```

Comment:
check that this selection does not include oil boilers

Reply:
Yes it's only oil-1980, oil-1990, ... , not oil boiler.